README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Go version](https://img.shields.io/badge/go-1.21-blue.svg)
 [![Go Reference](https://pkg.go.dev/badge/github.com/celestiaorg/nmt.svg)](https://pkg.go.dev/github.com/celestiaorg/nmt)
-![golangci-lint](https://github.com/celestiaorg/nmt/workflows/golangci-lint/badge.svg?branch=main)
+![golangci-lint](https://github.com/celestiaorg/nmt/workflows/lint/badge.svg?branch=main)
```

Comment:
Hmm if I go to that link it says no status. We could also get ride of the badge entirely 

![Screenshot 2024-05-13 at 2 54 49 PM](https://github.com/celestiaorg/nmt/assets/3699047/95ac3de0-28fb-4167-9360-c854be7e64e3)

```suggestion
```

Reply:
I am down with removing it entirely cc: @evan-forbes wdyt?

Reply:
I changed the url to use the default branch (removed the branch parameter which as the results uses the default branch of `main`). With this change the status badge is `failing`, which I think should become `passing` after this PR.